dapps/kiosk-cli/package.json

```diff
@@ -17,6 +17,6 @@
   "dependencies": {
     "@mysten/kiosk": "^0.9.34",
     "@mysten/sui": "^1.18.0",
-    "commander": "^11.0.0"
+    "commander": "^11.1.0"
```

Comment:
@hayes-mysten updated this based on ci error. Seems harmless but want to make sure.